crates/validator/src/db/schema.rs

```diff
@@ -0,0 +1,12 @@
+diesel::table! {
+    transactions (transaction_id) {
+        transaction_id -> Binary,
+        account_id -> Binary,
+        block_num -> BigInt,
+        initial_state_commitment -> Binary,
+        final_state_commitment -> Binary,
+        input_notes -> Binary,
+        output_notes -> Binary,
+        size_in_bytes -> BigInt,
+    }
+}
```

Comment:
This should be generated for us. Maybe I'm missing something from build.rs or maybe we need to add something to the Makefile